src/sentry/replays/query.py

```diff
@@ -637,13 +625,13 @@ def _activity_score():
     ),
     "count_segments": Function("count", parameters=[Column("segment_id")], alias="count_segments"),
     "count_errors": Function(
-        "uniqArray",
-        parameters=[Column("error_ids")],
+        "sum",
```

Comment:
TODO: may be worth trying out hyperloglog https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/uniqhll12/ but not important

Reply:
30% slower than counting.